src/constants/misc.js

```diff
@@ -254,26 +254,11 @@ export const ranks = {
     tag: "BUILD TEAM",
   },
 
-  MODERATOR: {
-    color: "2",
-    tag: "MOD",
-  },
-
   GAME_MASTER: {
     color: "2",
     tag: "GM",
   },
 
-  HELPER: {
-    color: "9",
-    tag: "HELPER",
-  },
-
-  "JR HELPER": {
-    color: "9",
-    tag: "JR HELPER",
-  },
-
```

Comment:
This is not related to this PR though...

Reply:
![image](https://user-images.githubusercontent.com/40838999/153779012-988ecfba-93ab-489b-9f5c-c41517751165.png)
![image](https://user-images.githubusercontent.com/40838999/153779004-1a79bb6c-7fbd-4e52-b291-1ce4babcbca9.png)